include/pybind11/chrono.h

```diff
@@ -161,9 +161,17 @@ template <typename Duration> class type_caster<std::chrono::time_point<std::chro
         // > If std::time_t has lower precision, it is implementation-defined whether the value is rounded or truncated.
         // (https://en.cppreference.com/w/cpp/chrono/system_clock/to_time_t)
         std::time_t tt = system_clock::to_time_t(time_point_cast<system_clock::duration>(src - us));
+
+        // std::localtime returns a pointer to a static internal std::tm object on success,
+        // or null pointer otherwise
+        // (https://en.cppreference.com/w/cpp/chrono/c/localtime)
+        std::tm *localtime_ptr = std::localtime(&tt);
```

Comment:
When can this fail? It would be nice to know when "otherwise" is the case. AFAIK, there haven't been issues with this, yet, but it's good to check, I suppose.

Reply:
found this on SO https://stackoverflow.com/a/58432714/3951920

Reply:
Thanks!